### DISCUSSION TITLE
Unable to build DBeaver in Eclipse (unknown extension point)

### DISCUSSION THREAD

**Discussion by deceits** [dbeaver/dbeaver#8671] [Q&A] [answered]
Hi, 
I am trying to build Dbeaver in Eclipse. Most plugins build are working fine except it failed on:

org.kiss.dbeaver.ext.ui.colortheme 
(Unknown extension point: 'org.github.eclipsecolortheme.mapper')  

org.jkiss.dbeaver.ui 
(Unknown extension point: 'com.github.eclipsecolortheme.mapper') 

any idea?

**Comment by serge-rider** [comment, marked as answer]
It is just a warning, you can ignore it (and configure Eclipse to ignore such type of errors)
